constants.py

```diff
@@ -443,3 +466,21 @@
 kCameraServoPGain = 0.15
 kCameraServoIGain = 0.0
 kCameraServoDGain = 0.0
+
+# Intake Camera
+kIntakeCameraMaxVerticalFOV = Rotation2d.fromDegrees(25)
+kIntakeCameraMinVerticalFOV = Rotation2d.fromDegrees(-25)
+kIntakeCameraMaxHorizontalFOV = Rotation2d.fromDegrees(35)
+kIntakeCameraMinHorizontalFOV = Rotation2d.fromDegrees(-35)
+kIntakeCameraHeightInMeters = 0.5
+kIntakeCameraCenterOffsetInMeters = 0.2
+kIsIntakeCameraCentered = False
+
+kSimIntakeCameraObjectName = "Intake Camera"
+
+# Intake auto intake constants
+kDriveToBallPGain = 0.5
```

Comment:
Out of curiosity how did these get determined?

Reply:
at the moment its a generic guess - needs to be tuned on the actual system